src/unity/python/turicreate/visualization/plot.py

```diff
@@ -107,13 +107,18 @@ def save(self, filepath, include_data=True):
 
         Parameters
         ----------
+        filepath: string
+            The destination filepath where the plot object must be saved as.
+            The extension of this filepath determines what format the plot will
+            be saved as. Currently supported formats are JSON, PNG, and SVG.
         include_data : bool, optional
             If True, save's the Plot in a vega spec with the data spec
-            included.
+            included. This parameter is not important when saving as a PNG/SVG
```

Comment:
This is really weird. I think we should get rid of this parameter entirely, instead. (When would the user want to save without data??).

Reply:
As discussed in person, removing the `include_data` flag from `save` but keeping it in the `_get_vega` and `_get_data` functions.